cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,314 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/gen/cloudwatch"
+	"github.com/awslabs/aws-sdk-go/gen/ec2"
+	"github.com/awslabs/aws-sdk-go/gen/elb"
+)
+
+const (
+	awsCPU                string = "aws.ec2.cpu"
+	awsNetwork            string = "aws.ec2.net.bytes"
+	awsEC2DiskBytes       string = "aws.ec2.disk.bytes"
+	awsEC2DiskOps         string = "aws.ec2.disk.ops"
+	awsStatusCheckFailed  string = "aws.ec2.status.failed"
+	awsELBLatencyMin      string = "aws.elb.latency.minimum"
+	awsELBLatencyMax      string = "aws.elb.latency.maximum"
+	awsELBLatencyAvg      string = "aws.elb.latency.average"
+	awsELBHostsHealthy    string = "aws.elb.hosts.healthy"
+	awsELBHostsUnHealthy  string = "aws.elb.hosts.unhealthy"
+	awsEC2CPUDesc         string = "The average CPU Utilization, gathered at a 60 second interval and averaged over five minutes."
```

Comment:
Desc goes first, so DescAWSEC2CPU ... kind of ugly in this case but that is our standard.